applications/matter_bridge/boards/nrf7002dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Set 10 BLE connections, as it is an upper limit supported caused by RAM usage due to using Thread and BLE at same time.
+CONFIG_BT_MAX_CONN=10
```

Comment:
Why do we need to set it to 10, as it is 10 by default (application's Kconfig)?

Reply:
I left here only LTO